processccd.rst

```diff
@@ -4,62 +4,66 @@
 ProcessCcdTask
 ##############
 
-ProcessCcd (which is a ``Cmd Line Task``, and thus can be run directly
-by typing processCcd.py at a shell prompt) executes the steps of how
-an image is processed from raw data to a science-grade images and catalogs that can
-be used in analyses.
+ProcessCcdTask (available as the ``processCcd.py`` ``command line
+task``) executes the steps of how an image is processed from raw
+uncorrected CCD-level data finally to science-grade images and
+catalogs.
 
-In more detail: ProcessCcd as a whole executes many of the functions
-that are in multiple packages in other astronomy analysis frameworks.
+It takes as input to its ``run`` method a single butler data reference
+for ``raw`` data and then outputs cleaned images (as a
+``lsst.afw.image.ExposureF`` field of a ``lsst.pipe.base.Struct``) and
+catalogs (``background`` and ``sourceCat`` fields of a
+``lsst.pipe.base.Struct``) to be used by later steps.
 
-- The initial step is to do ``Instrument Signature Removal`` through running :doc:`isrTask <isrtask>` to correct the images for all the issues involved in taking a raw image CCD through to a processed one (e.g. doing the bias and dark current corrections, flat-fielding, etc.).
+In more detail, ProcessCcdTask executes the following steps:
 
-- The second step is to do ``Image Characterization`` by running :doc:`characterizeImageTask <charimg>`), which includes for our purposes: object detection, repairing of cosmic ray defects, measuring and subtracting of sky background, and then finally measuring bright sources and using this to estimate background and PSF of an exposure.
+
+1.  ``Instrument Signature Removal`` -- Implemented by the :doc:`IsrTask <isrtask>` sub-task (isr configuration), this step removes CCD signatures (such as bias, dark current, flat-fielding, and cross-talk) and masks bad pixels.
+
+2. ``Image Characterization`` -- Implemented by the :doc:`CharacterizeImageTask <charimg>` sub-task (charImage configuration), this step repairs cosmic ray defects, estimates and subtracts a background, does object detection, and estimates a PSF.
```

Comment:
As with all my comments, I expect you to reword things.  I don't want you to use things verbatim.

I think "repair" gets the notion across too.  In this process, CRs are detected, masked and interpolated.